### PR TITLE
Edge auto register

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds additional information related to the registration tokens and the ability to auto-register edge devices. The Edge cluster information is also broken up to improve readability. 

💻 [Preview](https://deploy-preview-1129--docs-spectrocloud.netlify.app/clusters/edge)
🎫 [PE-854](https://spectrocloud.atlassian.net/browse/PE-854)

## Registration Token
Steps dedicated to the usage of a registration token have been added to the Register and Manager an Edge Cluster page.

CleanShot 2023-02-15 at 16 41 12

## Information Architecture Changes

Before:

CleanShot 2023-02-15 at 16 25 30

After:

CleanShot 2023-02-15 at 16 27 05

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1129#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1129#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by 3pings** [comment]
Are we merging "Edge" and "Edge-Native" in the docs?  As "Edge" which is really "Edge Virtualized (a GE Only thing)" is being hidden in the product it makes sense to me.  I like the look.

**Comment by karl-cardenas-coding** [comment]
> Are we merging "Edge" and "Edge-Native" in the docs? As "Edge" which is really "Edge Virtualized (a GE Only thing)" is being hidden in the product it makes sense to me. I like the look.

That is correct. Right now, edge virtualized is removed from the docs.

**Review by 3pings — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1129#pullrequestreview-1302097304" title="Approved">3pings</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1129#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:125]:
```suggestion
You can use several software tools to create a bootable USB drive, such as [balenaEtcher](https://www.balena.io/etcher). For a PXE server, there are open-source projects such as [Fog](https://fogproject.org/download) or [Windows Deployment Services](https://learn.microsoft.com/en-us/windows/deployment/wds-boot-support) for Windows.
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:130]:
```suggestion
To deploy the device and register it with Palette, refer to the [Register and Manage Edge Native Clusters](/clusters/edge/native) guide.
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:164]:
```suggestion
2. Use the content from the template in the [Set up Device section](/clusters/edge/native#setupdevice) to add your site configuration. Make the necessary changes for your environment.
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:184]:
```suggestion
This generates an ISO file called ci.iso in the current directory.
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:199]:
```suggestion
8. In the **Edge Hosts** tab, click **Download Palette Edge Installer**. This downloads the default Palette Edge Installer image (ISO).
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:196]: Karl, try combining this with the next step, as shown in line 199.

The optimum number of steps in a procedure is 7. Sometimes, you need more, but these steps combine well to reduce the number of overall steps.

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:208]:
```suggestion
11. Select other settings, such as Network, Datastore, or Folder as appropriate for your environment.
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:212]: Let's try combining steps 12 and 13.

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:214]:
```suggestion
13. Power on the VM and monitor the VM console for log messages. The installer and user data will be copied to the hard disk, and the VM will shut down.
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:220]:
```suggestion
15. Edit the VM settings and delete the two CD drives that were previously attached.
```

File: content/docs/04-clusters/03-edge/01.8-installer-image.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/01.8-installer-image.md:231]:
```suggestion
You can validate the OVA file by using the [ovftool](https://developer.vmware.com/web/tool/4.4.0/ovf) tool provided by VMware. To validate an OVA file using ovftool, follow these steps:
```

**Review by ritawatson — APPROVED** [review]
LGTM with some suggestions

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1129#pullrequestreview-1302097304" title="Approved">3pings</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1129#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1129#pullrequestreview-1302204651" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/03-edge.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/03-edge.md:45]: @ritawatson I added it to the team meeting agenda 📆

File: content/docs/04-clusters/03-edge/01-architecture.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/03-edge/01-architecture.md:72]: Probably the later.

File: content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/08-create-cluster.md:144]:
```suggestion
The deletion of an AWS cluster results in the removal of all instances and associated resources created for the cluster. To perform a cluster deletion, use the following steps. 
```
```suggestion
The deletion of an AWS cluster results in removing all instances and associated resources created for the cluster. To perform a cluster deletion, use the following steps. 
```

**Comment by spectro-prow** [comment]
@karl-cardenas-coding: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 8330ca95f442a450b1e0e467fc6b41d069cffa6d | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1129/librarium_build/1626371014193582080) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/03-edge/02-native.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/02-native.md:310]:
```suggestion
2. Navigate to the left **Main Menu** and click **Clusters**

```

File: content/docs/04-clusters/03-edge/02-native.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/02-native.md:313]:
```suggestion
3. Select the cluster you want to delete.

```

File: content/docs/04-clusters/03-edge/02-native.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/02-native.md:316]:
```suggestion
4. Click the **Settings drop-down Menu**.

```

File: content/docs/04-clusters/03-edge/02-native.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/02-native.md:319]:
```suggestion
5. Click **Delete Cluster**.

```

File: content/docs/04-clusters/03-edge/02-native.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/02-native.md:322]:
```suggestion
6. Type the name of the cluster and click **OK**

```

File: content/docs/04-clusters/03-edge/02-native.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/02-native.md:324]:
```suggestion
The cluster status is updated to **Deleting** while cluster resources are being deleted. When all resources are successfully deleted, the cluster status is updated to **Deleted** and the cluster is removed from the list. The delete operation returns the edge hosts to the **Ready** state. All the artifacts related to the Kubernetes distribution are removed. After the delete process, the edge hosts are available for deployment to a new cluster.

```

**Review by ritawatson — APPROVED** [review]
LGTM with comments in 02-native.md

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1129#pullrequestreview-1302097304" title="Approved">3pings</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1129#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1129#pullrequestreview-1302460724" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/03-edge/02-native.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/03-edge/02-native.md:76]: It's `stages` in the configuration file, so it's correct.

**Pull request merged**
